lib/runtime/LoadScriptRuntimeModule.js

```diff
@@ -91,7 +91,13 @@ class LoadScriptRuntimeModule extends HelperRuntimeModule {
 		]);
 
 		return Template.asString([
-			"var inProgress = {};",
+			`var inProgress = {}`,
+			`if (${RuntimeGlobals.global}) {`,
+			Template.indent([
+				`inProgress = ${RuntimeGlobals.global}.__WEBPACK_LOADER_PROGRESS__ =`,
```

Comment:
Using a global is not acceptable. webpack should avoid globals. This only causes multiple webpack builds on the same page to conflict. Maybe not now, but in future when this mechnism is changed or whatever.

The script tag is the only "global" you can use. And use also spec-conform things on it.

> `script.onerror = script.onload = onScriptComplete;`

That the offending line. Find a better way to do that when others have always set `onerror/load`.

Store the old value for overriding and call it in the handler.

or use `script.addEventListener`, but here there are a few questions about:
* browser support?
* `// avoid mem leaks in IE.` `script.onerror = script.onload = null;` what is the reason behind that and does using `addEventListener` break it?